buildscripts/CMakeLists.txt

```diff
@@ -442,7 +461,7 @@ set(OPENMW_PATCH
 )
 
 ExternalProject_Add(openmw
-	DEPENDS boost openal osg mygui ffmpeg sdl2 bullet
+	DEPENDS boost openal osg mygui ffmpeg sdl2 bullet lz4
```

Comment:
Is there a PR/MR which tracks the feature that requires lz4? I'll merge this once the feature is implemented.

Reply:
https://gitlab.com/OpenMW/openmw/-/merge_requests/350

Already merged, and I have have libopenmw.so building on gitlab so that we don't make any breaking changes upstream. That should help you guys. :)

Reply:
We'll eventually be able to support loading/saving in lz4, saving yet more space.